### PR TITLE
Create artado.json

### PR DESCRIPTION
**Pull request by Sxinar**

### CODE REVIEW

**Review by CuteDog5695 — CHANGES_REQUESTED** [review]
Why do you need NS records?

File: domains/artado.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "owner": {
+        "username": "sxinar",
+        "email": "oshidev@proton.me",
```
Comment (CuteDog5695) [inline domains/artado.json:4]:
```suggestion
        "email": "oshidev@proton.me"
```

File: domains/artado.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    },
+
+    "record": {
+        "NS": ["ns1.byet.org , ns2.byet.org , ns3.byet.org , ns4.byet.org , ns5.byet.org"],
```
Comment (CuteDog5695) [inline domains/artado.json:8]:
```suggestion
        "NS": ["ns1.byet.org , ns2.byet.org , ns3.byet.org , ns4.byet.org , ns5.byet.org"]
```

**Review by CuteDog5695 — CHANGES_REQUESTED** [review]
Missed these ones, anyways

**Comment by CuteDog5695** [comment]
Can't you do it via CNAME? Or am I wrong? (As far as I remember, its possible without NS records)

**Comment by Sxinar** [comment]
> Can't you do it via CNAME? Or am I wrong? (As far as I remember, its possible without NS records)

Yeah İTS possible because i want to vistapanel cname errors (i dont have screenshot) Ns better than

(İ have A very lazy English i am sorry)

**Review by CuteDog5695 — APPROVED** [review]
I guess you can have NS records then.... (Its for the reason earlier, not for the last comment)

**Review by CuteDog5695 — APPROVED** [review]

**Pull request merged**